cmd/runtimetest/main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/mount"
```

Comment:
On Thu, May 26, 2016 at 06:24:30PM -0700, 梁辰晔 (Liang Chenye) wrote:
> > +	"github.com/docker/docker/pkg/mount"
> 
> `docker/pkg/mount` supports different platform that is why I pull this dependency.
> If we just need to support Linux now, it could be a single function.

Hmm.  I think we *do* want to support multiple platforms (and stay as
platform-agnostic as possible here).  Anything that's just
conformance-testing for Linux namespaces should probably go in the
runC test-suite ;).  That probably means either pushing to get
docker/pkg/mount split out into it's own repo, or auditing it to see
how much unrelated stuff we're pulling in.